src/calendar-list/index.js

```diff
@@ -219,6 +223,7 @@ class CalendarList extends Component {
         initialScrollIndex={this.state.openDate ? this.getMonthIndex(this.state.openDate) : false}
         getItemLayout={this.getItemLayout}
         scrollsToTop={this.props.scrollsToTop}
+        initialNumToRender={this.props.horizontal ? this.getNumToRender(width, this.props.calendarWidth) : this.getNumToRender(height, this.props.calendarHeight)}
```

Comment:
Why not call for this calculation once in the constructor and use the value here instead of calling the method on every render (scroll)?